data/pokedex.ts

```diff
@@ -19937,4 +19967,124 @@ export const Pokedex: {[speciesid: string]: SpeciesData} = {
 		eggGroups: ["Undiscovered"],
 		newmeta: "true",
 	},
+	roedrunna: {
+		num: 10092,
+		name: "Roedrunna",
+		types: ["Light", "Psychic"],
+		baseStats: {hp: 75, atk: 61, def: 101, spa: 103, spd: 64, spe: 76},
+		abilities: {0: "Analytic", 1: "Keen Eye", H: "Stakeout"},
+		heightm: 0.3,
+		weightkg: 20,
+		color: "White",
+		eggGroups: ["Field"],
+		newmeta: "true",
+	},
+	maliceros: {
+		num: 10093,
+		name: "Maliceros",
+		types: ["Dark", "Ground"],
+		baseStats: {hp: 94, atk: 124, def: 114, spa: 64, spd: 74, spe: 64},
+		abilities: {0: "Truant", 1: "Weak Armor", H: "Swift Swim"},
+		heightm: 1.1,
+		weightkg: 80,
+		color: "Black",
+		eggGroups: ["Water 2"],
+		newmeta: "true",
+	},
+	breechon: {
+		num: 10094,
+		name: "Breechon",
+		types: ["Flying"],
+		baseStats: {hp: 65, atk: 70, def: 40, spa: 47, spd: 30, spe: 70},
+		abilities: {0: "Cloud Nine", 1: "Fur Coat", H: "Aerilate"},
```

Comment:
Breechon and Nimbichon are part of the same line, but you missed the evo & prevo stuff completely here.